.rubocop_todo.yml

```diff
@@ -0,0 +1,295 @@
+# This configuration was generated by
```

Comment:
Haste is the enemy of perfection. The rubocop generated the todo you must fix. In a giant project I would understand, but, it's a small project. Must be done.